### ISSUE TITLE
Do you need Filipino Translation for Octobercms?

### ISSUE THREAD

**Issue by josephbuarao** [octobercms/october#3424]
Hi there,

Good day and how are you? I hope this message finds you well.

I am Joseph Buarao , who has been doing high quality Filipino and Tagalog translations for several projects.

Consider this message as my letter of application. I am a web developer by profession and an experienced Filipino translator - these are on top of being an experienced web developer with more than 6 years of experience and specialized in the following scripting languages and technologies (HTML, CSS, JS, JQUERY, PHP, WORDPRESS and CONCRETE5). This means I know which codes are to be translated, and which ones that are to remain as it is. I am a strong advocate of thought-by-thought instead of word-by-word style of doing the translations, and I am carefully strict in grammars and spellings and proper word conjugations of my languages, Filipino and Tagalog.

I hope to hear from you soon and work with you in the translations. You can count on me that I will do my job with utmost dedication. Thank you and more power to you.

You can check my previous contributions:

https://github.com/textpattern/textpattern/commit/29308ff0d524e0b647ebb609710fdcb8acaee574
https://crowdin.com/project/textpattern-cms-textpacks/fil

Best Regards,
Joseph Buarao

**Comment by LukeTowers** [comment]
@josephbuarao if you'd be willing to translate it; go for it! We won't be paying for anything however as this is an open source project and we don't even have enough money to pay ourselves 😉

**Comment by josephbuarao** [comment]
Yes, I understand. It's open source and I am willing to contribute. Could
you please send me how to get started on the translation? Simple
instructions will do.

Thank you!!

**Comment by LukeTowers** [comment]
Language files are stored in modules/*/lang. See https://github.com/octobercms/october/pull/2808/files for an idea of what you need to do to setup a new locale and provide the files.

**Comment by LukeTowers** [comment]
Grab the latest copy of the English translations to get the latest keys that need to be translated, other languages are more than likely going to be out of date
